source/agora/crypto/Key.d

```diff
@@ -134,27 +137,30 @@ public struct PublicKey
     /// Uses Stellar's representation instead of hex
     public string toString () const @trusted nothrow
     {
-        ubyte[VersionWidth + PublicKey.sizeof + ChecksumWidth] bin;
+        scope(failure) assert(0);
```

Comment:
Why ?

Reply:
The `encodeBech32` was not `nothrow` in the previous code, so this is leftover. I removed it.